libs/scripts/preview/preview.css

```diff
@@ -0,0 +1,134 @@
+.hlx-preview-overlay {
+    z-index: 99;
+    position: fixed;
+    bottom: 32px;
+    right: 32px;
+    color: #eee;
+    font-weight: 600;
+    font-size: 24px;
+}
+
+.hlx-badge {
+    border-radius: 32px;
+    background-color: #888;
+    color: #eee;
+    padding: 16px 32px;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    position: relative;
+}
+
+.hlx-badge .hlx-open {
+    box-sizing: border-box;
+    position: relative;
+    display: block;
+    width: 22px;
+    height: 22px;
+    border: 2px solid;
+    border-radius: 100px;
+    margin-left: 16px;
+}
+.hlx-badge .hlx-open::after {
+    content: "";
+    display: block;
+    box-sizing: border-box;
+    position: absolute;
+    width: 6px;
+    height: 6px;
+    border-top: 2px solid;
+    border-right: 2px solid;
+    transform: rotate(-45deg);
+    left: 6px;
+    bottom: 5px;
+}
+
+.hlx-badge.hlx-testing {
+    background-color: #fa0f00;
+    color: #fff;
+}
+
+.hlx-popup {
+    position: absolute;
+    bottom: 64px;
+    right: 0;
+    background-color: #444;
+    min-width: 300px;
+    border-radius: 16px;
+    box-shadow: 0 0 10px #000;
+    font-size: 12px;
+}
+
+.hlx-popup a:any-link {
```

Comment:
Maybe `any-link` is not needed

Reply:
preview is non-functional and not used right now and is going to be worked on after this merge.